src/host/ft_host/precomp.h

```diff
@@ -32,3 +32,11 @@
 #include "common.hpp"
 
 #include "resource.h"
+
+#ifndef max
+#define max(a, b) (((a) > (b)) ? (a) : (b))
+#endif
+
+#ifndef min
+#define min(a, b) (((a) < (b)) ? (a) : (b))
+#endif

```

Comment:
Why are we defining these here and in propsheet's precomp?

Reply:
Because it's the only two spots they're needed and to signify they need to be cleaned up in #4152.

Reply:
Oh. That one got merged before this one. Then they should be gone.